packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -41,7 +41,7 @@ const initializableElements = ['atomic-search-interface', 'atomic-external'];
 
 /**
  * Retrieves `Bindings` on a configured parent search interface.
- * @param event Element on which to dispatch the event, which must be the child of a configured "atomic-search-interface" or "atomic-external" element.
+ * @param element Element on which to dispatch the event, which must be the child of a configured "atomic-search-interface" or "atomic-external" element.
  * @returns A promise that resolves on initialization of the parent "atomic-search-interface" or "atomic-external" element, and rejects when it's not the case.
  */
 export const initializeBindings = (element: Element) =>
```

Comment:
I would also update the type of `initialize` in `InitializableComponent` from `initialize?: () => void;` to `initialize?: () => void | Promise<void>;`